src/utils/ParameterConstraintsUtils.js

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+const operatorsDict = {
+  date: ['>', '<', '>=', '<=', '==', '!='],
+  number: ['>', '<', '>=', '<=', '==', '!='],
+  int: ['>', '<', '>=', '<=', '==', '!='],
+  string: ['==', '!='],
+  bool: ['==', '!='],
+};
+
+const getIsVarTypesComparisonValid = (varType, varTypeToCompare) => {
+  const numberVarTypes = ['int', 'number'];
+  return varType === varTypeToCompare || [varType, varTypeToCompare].every((el) => numberVarTypes.includes(el));
+};
+
+const getParameterValidationConstraint = (validationString, varType, parametersList) => {
+  if (validationString) {
```

Comment:
Do you realy want to test if _validationString_ is empty ?
In the case where you only want to test if _validationString_ is _null_ or _undefined_, preferably use _!= null_ test.

Reply:
I prefer test it in case there is an arror in solution configuration (something like: `validation: ''`) :slightly_smiling_face: